static/app/views/performance/transactionSummary/transactionAnomalies/anomaliesTable.tsx

```diff
@@ -79,11 +82,36 @@ function renderBodyCellWithMeta(location: Location, organization: Organization)
         </ConfidenceCell>
       );
     }
+    if (column.key === 'expected') {
+      return (
+        <NumberCell>
+          <Count value={dataRow.expected} />
+        </NumberCell>
+      );
+    }
+    if (column.key === 'received') {
+      return (
+        <NumberCell>
+          <Count value={dataRow.received} />
+          <IconArrow
+            size="sm"
+            direction={dataRow.received > dataRow.expected ? 'up' : 'down'}
```

Comment:
should an arrow be shown if `dataRow.received == dataRow.expected`?

Reply:
By virtue of the fact it's an anomaly I don't think that condition is possible if the api is working correctly, if it's that close I think it's fine still showing an arrow, not that worried about it.